gift_card_test.go

```diff
@@ -0,0 +1,153 @@
+package goshopify
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+)
+
+func TestGiftCardGet(t *testing.T) {
+	setup()
+	defer teardown()
+
+	httpmock.RegisterResponder(
+		"GET",
+		fmt.Sprintf("https://fooshop.myshopify.com/%s/gift_cards/1.json", client.pathPrefix),
+		httpmock.NewBytesResponder(
+			200,
+			loadFixture("gift_card/get.json"),
+		),
+	)
+
+	giftCard, err := client.GiftCard.Get(1)
+	if err != nil {
+		t.Errorf("GiftCard.Get returned error: %v", err)
+	}
+
+	expected := GiftCard{ID: 1}
+	if expected.ID != giftCard.ID {
```

Comment:
Simplify:
`giftCard.ID == 1`

And maybe check 2-3 more fields, just in case.

same for all the others below (simplify the `if`)